style.css

```diff
@@ -0,0 +1 @@
+.my-style { color: tomato; }
```

Comment:
According to the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) please let's add a valid CSS color property to the `h1` tag.
`A .css file that includes one class with styles for your header (make it in your favorite color).` 

👇 This is not a valid color 😀, kindly fix
```
.my-style { color: tomato; }
```